recipe/meta.yaml

```diff
@@ -46,7 +46,7 @@ requirements:
     - qtpy >=1.5.0
     - rope >=0.10.5
     - sphinx
-    - spyder-kernels >=0.1,<1
+    - spyder-kernels >=0.4.2,<1
```

Comment:
This is the most important change of this release in terms of Spyder's dependencies.